x/gov/keeper.go

```diff
@@ -8,18 +8,11 @@ import (
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/cosmos/cosmos-sdk/x/gov/types"
 	"github.com/cosmos/cosmos-sdk/x/params"
+	"github.com/cosmos/cosmos-sdk/x/supply"
```

Comment:
I don't think we should be aliasing types in `x/{module}/exported`. This should be `github.com/cosmos/cosmos-sdk/x/supply/exported`